src/app/pages/OnDemandTvPage/index.test.jsx

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
+import clone from 'ramda/src/clone';
```

Comment:
```suggestion
import assocPath from 'ramda/src/assocPath';
```
if you wanted to do this: https://github.com/bbc/simorgh/pull/6710/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx#r433365470